lib/chef-dk/command/generator_commands.rb

```diff
@@ -69,6 +69,12 @@ module SharedGeneratorOptions
         :description => "Use to set arbitrary attribute KEY to VALUE in the code_generator cookbook",
         :default => [],
         :proc => Proc.new { |s| generator_args <<  s.split("=").map(&:strip) }
+
+      option :pipeline,
+        :short => "-f PIPELINE",
```

Comment:
@afiune explained to me that this actually aligns with other symantics and is more clear in the context of workflow so I'm good in the hood as they say.